ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/StandardComponentInitializer.java

```diff
@@ -497,7 +498,11 @@ public void initialize() {
         loaderToolbarGroup.add(loaderAction);
 
         // Define hot-keys
-        keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode('n').build(), "navigateToFile");
+        if(UserAgent.isMac()){
```

Comment:
In this case it would be better to write something like this:

int charCode = UserAgent.isMac() ? 'N' : 'n';
keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode(charCode).build(), "navigateToFile");

I think it will be more readable and understandable.
WDYT, guys?